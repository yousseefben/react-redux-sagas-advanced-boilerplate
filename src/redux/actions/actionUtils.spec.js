import { createAction, createActionTypes } from './actionsUtils';

describe('action utils', () => {
  describe('action type creator', () => {
    it('should return action type', () => {
      expect(createActionTypes('TEST', ['GET', 'POST'])).toMatchSnapshot();
    });
  });
  describe('action creator', () => {
    it('should return action', () => {
      expect(createAction('TEST', { a: '1' })).toMatchSnapshot(1);
    });
  });
});
