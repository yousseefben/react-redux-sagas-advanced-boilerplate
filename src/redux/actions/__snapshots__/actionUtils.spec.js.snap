// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action utils action creator should return action 1`] = `
Object {
  "payload": Object {
    "a": "1",
  },
  "type": "TEST",
}
`;

exports[`action utils action type creator should return action type 1`] = `
Object {
  "GET": "TEST_GET",
  "POST": "TEST_POST",
}
`;
